doc/Command-Reference.md

```diff
@@ -10397,6 +10397,38 @@ This command removes all unused images from the device, leaving only the current
   No image(s) to remove
   ```
 
+**sonic-install get-fips**
```

Comment:
```suggestion
**sonic-installer get-fips**
```
Fix typo to match the previous command headings?

Reply:
@mdanish-kh , thanks for your review, fixed.